deno/gateway/v10.ts

```diff
@@ -2051,6 +2053,33 @@ export interface GatewayPresenceUpdateData {
  */
 export type GatewayActivityUpdateData = Pick<GatewayActivity, 'name' | 'state' | 'type' | 'url'>;
 
+export interface GatewayMessagePollVoteData {
+	/**
+	 * ID of the user
+	 */
+	user_id: Snowflake;
+	/**
+	 * ID of the channel
+	 */
+	channel_id: Snowflake;
+	/**
+	 * ID of the message
+	 */
+	message_id: Snowflake;
+	/**
+	 * ID of the guild
+	 */
+	guild_id?: Snowflake;
+	/**
+	 * ID of the answer
+	 */
+	answer_id: number;
+}
+
+export type GatewayMessagePollVoteAdd = GatewayMessagePollVoteData;
+
+export type GatewayMessagePollVoteRemove = GatewayMessagePollVoteData;
```

Comment:
These need to be added to GatewayDispatchPayload and follow the structure of all other payloads